src/app/bugs/bugs.component.ts

```diff
@@ -1,15 +1,90 @@
-import { Component, OnInit } from '@angular/core';
+import {Component, OnInit, ViewChild} from '@angular/core';
+import {MatPaginator} from '@angular/material/paginator';
+import {MatSort} from '@angular/material/sort';
+import {MatTableDataSource} from '@angular/material/table';
+import { element } from 'protractor';
```

Comment:
I think the import for element may have been an auto import by VS Code.  It should be safe to remove on your next commit.